### ISSUE TITLE
Disabling "Allow subtitle extraction on the fly" broken

### ISSUE THREAD

**Issue by platinum55** [jellyfin/jellyfin#1163]
**Describe the bug**
I disabled the option "Allow subtitle extraction on the fly" to prevent jellyfin from spawning a ffmpeg process to extract subtitles on the fly. Sadly it does spawn the process even if the option is disabled.

As I use network device, the subtitle extraction process saturates the line and the transcoding process cannot go on, so the playback is stuck.

**To Reproduce**

1. Go to 'Transcoding'
2. Click on 'Allow subtitle extraction on the fly' to disable the option
3. Start a movie with subtitles
4. See two processes, one for subtitle extraction and one for transcoding

**Expected behavior**
spawn only one ffmpeg(transcoding) process when "Allow subtitle extraction on the fly" is disabled, hardcode subtitles if needed.

**System:**
 - OS: Docker
 - Browser: Chromium
 - Jellyfin Version: 10.2.2

**Comment by cvium** [comment]
I've been meaning to open an issue for it a few months ago... The config option is never used anywhere.

**Comment by cvium** [comment]
Go away bot

**Comment by sachk** [comment]
Can confirm this still happens on 10.4.3

**Comment by Artiume** [comment]
What should happen for Image based subtitles such as VobSub that attempts to play in a container or client that doesn't support the subtitles? One of the current issues is ts and mp4 have very limited subtitle support. mp4 natively supports VobSub so it might benefit us to look into DASH protocol to enable mp4 segments. I've been looking into how to stream mkv to allow native ASS support but the best I can think of is to fake the MIME type using video/webm but I don't know how well that would go over.

**Comment by cvium** [comment]
The checkbox still doesn't do anything in 10.7.

**Comment by Airyzz** [comment]
Any progress on this?

**Comment by jdenicola** [comment]
I Can confirm it's still not working on 10.8.3

**Comment by KarkaLT** [comment]
It is still happening on 10.10.7.

**Comment by eliliam** [comment]
Still appears to be a thing in 10.11.2

**Comment by cvium** [comment]
Yes, this is an open issue. You don't need to keep mentioning it. When it is fixed the issue will be closed

**Comment by ljohnso16** [comment]
Bump in latest version
